lib/lrama/grammar/binding.rb

```diff
@@ -30,6 +34,17 @@ def parameter_to_arg(symbol)
         end
         arg
       end
+
+      def token_to_args_s_values(token)
+        token.args.flat_map do |arg|
```

Comment:
Note: `Array#join` expands array of array recursively then both `#flat_map` and `#map` works. However returning unnested array is simpler.